### DISCUSSION TITLE
connection failing when connecting to newer ssh server

### DISCUSSION THREAD

**Discussion by jrishel** [ParallelSSH/parallel-ssh#396] [General]
I'm using ParallelSSH on a RHEL7 host (python 3.6, sourcing ParallelSSH 2.12.0 from pypi.  It works connecting to my RHEL7 and RHEL8 hosts, but on RHEL9, the connection fails. at the OS level, I can use SSH keys to connect without issue.  the error the RHEL9 ssh server indicates is:
` sshd[986124]: userauth_pubkey: key type ssh-rsa not in PubkeyAcceptedAlgorithms [preauth]`

Is there a way to tell pssh.clients to start using newer algo keys?  I'd rather update that than go to my RHEL9 hosts and tell them to start accepting legacy algo keys?  I did test that setting RHEL9 to legacy crypto policy lets the SSHClient connect, but that's not ideal.

**Comment by pkittenis** [comment]
Thanks for the interest.

Latest client supports newer keys.
